recipe/meta.yaml

```diff
@@ -12,10 +12,11 @@ package:
 source:
   fn: {{ name }}-{{ version }}.{{ bundle }}
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ bundle }}
-  {{ hash_type }}: {{ hash_val }}
+  {{ hash_type }}: {{ hash }}
 
 
 build:
+  noarch: python
   entry_points:
     - faker=faker.cli:execute_from_command_line
```

Comment:
As per upstream issue ( https://github.com/conda/conda-build/issues/2378 ), please add spaces on either side of `=`.